File: beecrowd/beecrowd-2862/becrowd2862.java
import java.util.Scanner;

public class becrowd2862 {
    public static void main(String[] args) {
        int n, p;
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        for (int i = 0; i < n; i++) {
            p = sc.nextInt();
            if (p > 8000) {
                System.out.println("Mais de 8000!");
            } else {
                System.out.println("Inseto!");
            }
        }
    }
}

File: beecrowd/beecrowd-2862/becrowd-2862.cpp
#include <iostream>

int main (int argc, char *argv[])
{
    int n, p;
    std::cin >> n;
    for (int i = 0; i < n; i++) {
        std::cin >> p;
        if (p > 8000) {
            std::cout << "Mais de 8000!\n";
        } else{
            std::cout << "Inseto!\n";
        }
    }
    return 0;
}
